web/src/client/users.js

```diff
@@ -64,13 +64,12 @@ class UsersBaseClient {
    * @return {Promise<User>}
    */
   async getUser() {
-    const user = await this.client.get("/users/first");
-
-    if (user === null) {
+    const response = await this.client.get("/users/first");
+    if (!response.ok) {
+      console.log("Failed to get first user config: ", response);
```

Comment:
I think `console.error` would be better here...

Reply:
agreed